public/static/js/subscriptions.js

```diff
@@ -341,11 +347,11 @@ function addProjectSelectors(subscription, project) {
 }
 
 function validateDuration(duration) {
-  if (!Number.isInteger(+duration)) {
-    return duration + " must be an integer";
+  if (duration === "" || !Number.isInteger(+duration)) {
+    return "A duration must be an integer";
```

Comment:
What was the reason for changing this message? I feel like it's more vague now since it's not clear what field is invalid